rs/registry/canister/src/get_node_operators_and_dcs_of_node_provider.rs

```diff
@@ -20,11 +22,16 @@ impl Registry {
         )> = vec![];
         for (key, values) in self.store.iter() {
             if key.starts_with(NODE_OPERATOR_RECORD_KEY_PREFIX.as_bytes()) {
```

Comment:
Probably the get_key_family functions would be better here, so they can deal with the dechunking maybe.